pymatgen/io/vasp/optics.py

```diff
@@ -110,7 +112,8 @@ def from_directory(cls, directory: Path | str):
         """Construct a DielectricFunction from a directory containing vasprun.xml and WAVEDER files."""
         d_ = Path(directory)
         vrun = Vasprun(d_ / "vasprun.xml")
-        waveder = Waveder.from_binary(d_ / "WAVEDER")
+        is_gamma = "gamma" in vrun.generator["subversion"]
```

Comment:
This is kinda hacky but the "gamma-only" is defined in `src/version.F` so I think we are OK.